### ISSUE TITLE
[Support] Repentance

### ISSUE THREAD

**Issue by Romz24** [Meowlala/BOITrueCoop#3]
Hello, are you planning to update this mod for the Repentance version?

**Comment by Meowlala** [comment]
No, Repentance supports True Co-op natively. Some features such as the multiplayer versus screen may come back in a separate mod.
